media/system/js/share.js

```diff
@@ -0,0 +1,31 @@
+jQuery(document).ready(function(e){e("#sharebutton").click(function()
```

Comment:
This file must be compressed. You can use jscompress.com to compress the content.